src/data/MetaDataItem.py

```diff
@@ -27,4 +27,15 @@ def to_json_str(self) -> str:
     def to_file(self):
         # Write the output to disk
         with open(self.id + '_metadata.json', 'w') as outfile:
-            json.dump(self.to_json, outfile, sort_keys=True, indent=2)
\ No newline at end of file
+            json.dump(self.to_json, outfile, sort_keys=True, indent=2)
+
+
+def create_metadata(id: str, variables: dict) -> MetaDataItem:
+    metadata = MetaDataItem(id,
+                            variables['title'],
+                            variables['url'],
+                            variables['collision_type'],
+                            variables['description'],
+                            variables['location'])
+    return metadata
```

Comment:
We don't need this function since we can essentially replicate this using the expansion operator
```python3
obj = { 'id': 1, 'title': 'title', 'url': 'url', 'collision_type': 'ctype', 'description': 'desc', 'location': 'loc'}
item = MetaDataItem(**obj)
```

Reply:
Cool, did know that was a part of python, I'll make those changes.